nestkernel/nest_time.h

```diff
@@ -387,7 +387,13 @@ class Time
   bool
   is_neg_inf() const
   {
-    return tics == LIM_NEG_INF.tics;
+    return tics <= LIM_NEG_INF.tics;
```

Comment:
@apeyser Correct, where 
 - "leave as is" means `<=`, 
- "remove inequality" means `==`,
- and "replace the inequality portion" means `assert(tics >= ...); return ... == ...;`
I would want to avoid the assertion for performance reasons, so maybe we should keep the code as it is currently in master, where it is actually `==` ( @jakobj Sorry for the back-and-forth!).